src/redux/features/editProfileSlice.ts

```diff
@@ -154,9 +154,8 @@ export const updateUser = createAsyncThunk(
     { rejectWithValue }
   ) => {
     try {
-      const response = await userService.updateUser(userId, params)
-      // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-      return response.data
+      await userService.updateUser(userId, params)
+      return true
```

Comment:
```suggestion
      await userService.updateUser(userId, params)
```

I don't think we need ```return true``` here. For what needs? We don't have to return true because you call the update function and it shouldn't return nothing. Or I am mistaken?

Reply:
My intention was to return `true` as a successful ending of `updateUser`, and for `fulfilled state. 
But if you think it is unnecessary, I will test updating without `true`.

Reply:
It will update user in successful ending. In case you get an error, ```catch``` block catchs it and returns with ```rejectWithValue```

Reply:
You're right, that's true. Changed